dev/benchmarks/macrobenchmarks/lib/main.dart

```diff
@@ -136,10 +138,10 @@ class HomePage extends StatelessWidget {
             },
           ),
           RaisedButton(
-            key: const Key(kImageFilteredTransformAnimationRouteName),
```

Comment:
Adding a benchmark should never involve deleting code so this diff should have been a red flag during the reviews...

Reply:
This is not intended. Some how when I copy paste things I override it :( I added it to a new patch #60412

Reply:
I've already fixed this in https://github.com/flutter/flutter/commit/2962912d68fb72d6c9fd60f652e27f3593d692d6

It has already been pushed to the main tree.